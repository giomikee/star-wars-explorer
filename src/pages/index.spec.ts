import { shallowMount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import Homepage from './index.vue'

const createWrapper = () => shallowMount(Homepage)

describe('Given Homepage component', () => {
  describe('When component is rendered', () => {
    it('Then it should match the snapshot', () => {
      const { element } = createWrapper()

      expect(element).toMatchSnapshot()
    })
  })
})
