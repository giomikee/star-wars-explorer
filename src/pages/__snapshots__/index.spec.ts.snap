// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Given Homepage component > When component is rendered > Then it should match the snapshot 1`] = `<span />`;
